src/Microsoft.SymbolStore.Client/StoreQueryBuilder.cs

```diff
@@ -17,14 +19,29 @@ internal static class StoreQueryBuilder
 
         public static readonly string SourcePrefix = "sha1";
```

Comment:
This was in the original code Tomas checked in.  I'm going to leave it alone for now, and revisit when I actually go to use it.